modern/src/DevicesList.js

```diff
@@ -44,11 +110,9 @@ const DeviceRow = ({ data, index, style }) => {
               <img className={classes.icon} src={`images/icon/${item.category || 'default'}.svg`} alt="" />
             </Avatar>
           </ListItemAvatar>
-          <ListItemText primary={item.name} secondary={item.uniqueId} />
+          <ListItemText primary={item.name} secondary={<OnlineStatus status={item.status} />} />
           <ListItemSecondaryAction>
-            <IconButton onClick={(event) => onMenuClick(event.currentTarget, item.id)}>
```

Comment:
How do we edit/remove devices now?

Reply:
At present, we can't. In the design, there are buttons to edit and delete devices on the pop-up screen.

Reply:
We need to keep it functional. We can't remove features.